lib/Serializer.php

```diff
@@ -69,8 +70,4 @@ public function serialize($data) {
 	public function deserialize($data, $type) {
 		return $this->serializer->deserialize($data, $type, 'json', []);
 	}
-
-	public function setIgnoredAttributes($attributes = []) {
```

Comment:
`setIgnoredAttributes` is no longer part of Symfony 5.

The new way is to use `AbstractNormalizer::IGNORED_ATTRIBUTES` - see code changes in `writelnToStream` below.